### ISSUE TITLE
Cards grid

### ISSUE THREAD

**Issue by btzr-io** [btzr-io/jelly-beats#202]
Improve the current grid to display cards:
>  Would recommend setting a min-width to the '.grid' div so the layout won't look broken if the window is too small. Also, a 'justify-content: center' for when the max window width is reached.

See: https://github.com/btzr-io/jelly-beats/pull/196

**Comment by chrisdemars** [comment]
Is this still an issue @btzr-io ?

**Comment by btzr-io** [comment]
@chrisdemars Yes, but we should probably redesign the card component first: https://github.com/btzr-io/jelly-beats/issues/203
